images/osm2pgsql/requirements.txt

```diff
@@ -1,2 +1,3 @@
 psycopg2
-mapcoloniesjsonlogger==0.0.14
\ No newline at end of file
+mapcoloniesjsonlogger==0.0.14
+requests
```

Comment:
let's stick to specific versions.
`2.8.6` for `pycopg` (latest so far)
`2.24.0` for `requests` (latest so far)

Reply:
fixed